CMakeLists.txt

```diff
@@ -196,8 +196,8 @@ set(_acts_autodiff_version 0.6)
 set(_acts_boost_version 1.71.0)
 set(_acts_dd4hep_version 1.21)
 set(_acts_edm4hep_version 0.7)
-set(_acts_geomodel_version 4.6.0)
 set(_acts_eigen3_version 3.3.7)
+set(_acts_geomodel_version 6.0.0)
```

Comment:
@dimitra97 is gonna update that to 6.2.0.

Reply:
Ah ok I didn't see that 6.2.0 was available. I will update that also.